src/fs/manifest/schema.rs

```diff
@@ -156,41 +150,85 @@ impl ConfigSection {
 #[serde(rename_all = "kebab-case")]
 pub enum MonitorConfig {
     AwsCloudwatch(AwsCloudwatch),
-    CloudflareObservability(CloudflareObservabilityConfig),
+    CloudflareObservability(CloudflareConfig),
 }
 
 impl MonitorConfig {
-    async fn load_monitor(&self, args: &RunSubcommand) -> Result<BoxedMonitor> {
+    async fn load_monitor(
+        &self,
+        args: &RunSubcommand,
+        ingress: &BoxedIngress,
+    ) -> Result<BoxedMonitor> {
         match self {
-            MonitorConfig::AwsCloudwatch(aws_cloudwatch) => aws_cloudwatch.load_monitor(args).await,
+            MonitorConfig::AwsCloudwatch(aws_cloudwatch) => {
+                aws_cloudwatch.load_monitor(args, ingress).await
+            }
             MonitorConfig::CloudflareObservability(cloudflare_observability) => {
-                cloudflare_observability.load_monitor(args).await
+                cloudflare_observability.load_monitor(args)
             }
         }
     }
 }
 
-impl Default for MonitorConfig {
-    fn default() -> Self {
-        Self::AwsCloudwatch(AwsCloudwatch::default())
-    }
-}
-
 #[derive(Clone, Default, Deserialize, Serialize, PartialEq, Debug)]
 pub struct AwsCloudwatch {}
 
 impl AwsCloudwatch {
-    async fn load_monitor(&self, _args: &RunSubcommand) -> Result<BoxedMonitor> {
-        todo!();
+    fn load_gateway_name(&self, args: &RunSubcommand, ingress: &BoxedIngress) -> Result<String> {
+        let gateway_name = match ingress.get_config() {
+            backend::IngressConfig::AwsRestApiGateway {
+                gateway_name,
+                region: _,
+                stage_name: _,
+                resource_path: _,
+                resource_method: _,
+            } => gateway_name,
+            _ => return Err(miette!("Ingress is not an AWS API Gateway.")),
+        };
+
+        let gateway_name = args
+            .aws_gateway_name()
+            .map(ToString::to_string)
+            .unwrap_or_else(|| gateway_name);
+
+        Ok(gateway_name)
     }
-}
 
-#[derive(Clone, Default, Deserialize, Serialize, PartialEq, Debug)]
-pub struct CloudflareObservabilityConfig {}
+    fn load_stage_name(&self, args: &RunSubcommand, ingress: &BoxedIngress) -> Result<String> {
+        let stage_name = match ingress.get_config() {
+            backend::IngressConfig::AwsRestApiGateway {
+                stage_name,
+                region: _,
+                gateway_name: _,
+                resource_path: _,
+                resource_method: _,
```

Comment:
Pretty sure there's a shorthand syntax for this...

```suggestion
                stage_name,
                ..
```